Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,27 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            // To do: call your implementation.
```

Comment:
Comments can be removed after implementation.

Reply:
Removed.